bin/duniverse.ml

```diff
@@ -123,6 +127,21 @@ let opam_cmd =
       $ remotes_t $ setup_logs () )),
     Term.info "opam" ~doc ~exits ~man )
 
+let opam_install_cmd =
+  let doc = "install packages that are not dune-compatible via opam" in
```

Comment:
How about `not duniverse-compatible` as now this also installs packages we don't know how to clone which might be built with dune?

Reply:
Same goes for the man section below of course!

Reply:
Yes I agree